package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mkdocs-ponylang",
-  "version": "0.2.6",
+  "version": "0.2.8",
```

Comment:
is this the version of our package? any idea why if that is a yes, that is is being changed here? is ours wrong and that is the 0.2.8 is the latest version of our mkdocs theme?

Reply:
Seems like `0.2.8` is indeed the latest version of our mkdocs theme, we just forgot to update this file when we bumped the version on `package.json`.